src/parts/Movies/Movie/Movie.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import GenreList from './GenreList';
+import ReleaseYear from './ReleaseYear';
+import Image from '../../../components/Image';
+import DotsVerticalIcon from 'mdi-react/DotsVerticalIcon'
+import Popup from '../../../components/Popup/Popup';
+import MovieMenu from './MovieMenu';
+import MovieForm from '../../MovieForm/MovieForm';
+import Confirm from '../../../components/Confirm';
+import useModal from '../../../hooks/useModal';
+
+const Movie = React.memo(props => {
+  const [showPopup, setShowPopup] = useState(false);
+  const { modal, openModal } = useModal();
+
+  const openEditModal = () => {
+    const modalBody = <MovieForm formTitle="EDIT MOVIE" formData={props.movie} onSubmit={() => alert('Movie Edited')} />;
+    openModal(modalBody, 'MovieFormModal');
+  };
+
+  const openConfrim = () => {
+    const confirm = (
+      <Confirm onConfirm={() => alert('MOVIE DELETEd')}>
+        <h3>DELETE MOVIE</h3>
+        <p>Are you sure you want to delete this movie?</p>
+      </Confirm>
+    );
+    openModal(confirm, 'ConfirmModal');
+  }
+
+  const handleOnMouseEnter = () => setShowPopup(true);
+  const handleOnMouseLeave = () => setShowPopup(false);
+
+  const { movie, viewMovieDetails } = props;
```

Comment:
destructure fejlécben

Reply:
Fixed in [0cd3b29](https://github.com/rkasza/netflix-roulette/pull/5/commits/0cd3b297078b39a413f47a5afd50d0985f364f48)